tmx_property.go

```diff
@@ -103,3 +107,24 @@ func (p Properties) GetFloat(name string) float64 {
 	}
 	return 0
 }
+
+func (p Properties) GetColor(name string) color.Color {
```

Comment:
please do not add mutliple unrelated changes/features into single PR

Reply:
No problem.